src/Components/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { ErrorPage } from "@boomerang-io/carbon-addons-boomerang-react";
+import EmptyGraphic from "./EmptyGraphic";
+
+const EmptyState: React.FC<any> = (props) => {
+  return <ErrorPage graphic={<EmptyGraphic />} {...props} />;
```

Comment:
If we are mainly setting `header` to null and have the same empty search message in multiple places do we want to set those as the default props?